website/docs/install.md

```diff
@@ -0,0 +1,124 @@
+---
+id: install
+title: Installation
+---
+
+## Prerequisites
+
+### Minimum Kubernetes Version
+
+**To use Gatekeeper, you should have a minimum Kubernetes version of 1.14, which adds
+webhook timeouts.**
+
+You can install Gatekeeper in earlier versions of Kubernetes either by
+removing incompatible fields from the manifest or by setting `--validate=false`
+when applying the manifest. Be warned that, without timeouts on the webhook, your
+API Server could timeout when Gatekeeper is down. Kubernetes 1.14 fixes this issue.
+
+### RBAC Permissions
+
+For either installation method, make sure you have cluster admin permissions:
+
+```sh
+  kubectl create clusterrolebinding cluster-admin-binding \
+    --clusterrole cluster-admin \
+    --user <YOUR USER NAME>
+```
+
+## Installation
+
+### Deploying a Release using Prebuilt Image
+
+If you want to deploy a released version of Gatekeeper in your cluster with a prebuilt image, then you can run the following command:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper/master/deploy/gatekeeper.yaml
```

Comment:
should this be pointing to master, given that it's not currently being updated?

Reply:
updated to release-3.1